app/collins/models/RemoteAssetFinder.scala

```diff
@@ -276,7 +276,7 @@ object RemoteAssetFinder {
   /**
    */
   def apply(clients: Seq[RemoteAssetClient], pageParams: PageParams, searchParams: AssetSearchParameters): (Seq[AssetView], Long) = {
-    val key = searchParams.paginationKey + clients.map { _.tag }.mkString("_")
+    val key = searchParams.paginationKey.getOrElse("") + clients.map { _.tag }.mkString("_")
     val stream = Cache.getAs[RemoteAssetStream](key).getOrElse(new RemoteAssetStream(clients, searchParams))
     val results = stream.slice(pageParams.page * pageParams.size, (pageParams.page + 1) * (pageParams.size))
     Cache.set(key, stream, 30)
```

Comment:
I don't really see a reason to change it right now, it's been a while since we actually used the multi-collins functionality in production so i suggest keeping it at 30 to see how it works out. I'll open a ticket to move it to a configuration parameter though.